### DISCUSSION TITLE
Support for comment shortcut (i.e. Ctrl+/) and specific function syntax

### DISCUSSION THREAD

**Discussion by nanakinoir** [hoppscotch/hoppscotch#4506] [Ideas]
Being asked to try out Hoppscotch and I noticed 2 issues:

1. Hoppscotch does not seem to support the common shortcut keys "Ctrl + /" for toggling comments in the scripting editor.
2. It's giving me a semantic error of "Invalid argument: cannot convert from Function.prototype to string" for the following code snippet

```
function simpleTest() {
    return 'liGHt-HouSE'.replace(/[A-Za-z]/g, function(c) {
        return c.match(/[A-Z]/) ? '1' : '0';
    });
}

console.log(simpleTest());
```

The above code seems to work elsewhere.

Should I raise a bug for these items?

**Comment by jamesgeorge007** [comment]
Hi @nanakinoir, could you try the above after opting in to the new scripting experience (`Experimental scripting sandbox` opted in via settings)?

<img width="2970" height="1058" alt="image" src="https://github.com/user-attachments/assets/816784d4-afd1-4c8b-a725-ea822c12c984" />
